moveit_ros/planning_interface/moveit_cpp/src/moveit_cpp.cpp

```diff
@@ -57,9 +57,9 @@ MoveItCpp::MoveItCpp(const ros::NodeHandle& nh, const std::shared_ptr<tf2_ros::B
 {
 }
 
-MoveItCpp::MoveItCpp(const Options& options, const ros::NodeHandle& /*unused*/,
+MoveItCpp::MoveItCpp(const Options& options, const ros::NodeHandle& nh,
```

Comment:
Is this related in any way? Also, the other constructor doesn't seem to initialize `node_handle_` this way

Reply:
Yes, this node handle will be passed to planning components to load the parameters (the parameters are under MoveItCpp namespace